### PR TITLE
Fix go testgen

### PR DESCRIPTION
**Pull request by tadelesh**
When SDK version is bigger than v1, Go will use package/vX to represent the import. This PR fix the import issue of SDK module for such condition.

### CODE REVIEW

**Review by lirenhe — COMMENTED** [review]
Do you have test case to cover your change?

**Comment by tadelesh** [comment]
> Do you have test case to cover your change?

One of the integrate test case will cover the change.

File: tools/sdk-testgen/packages/autorest.gotest/test/integrationtest/testAutorest.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ async function runAutorest(readmePath: string, extraOption: string[]) {
     });
 }
 function getExtraOption(outputFolder: string) {
-    return [`--output-folder=${outputFolder}`, '--use=@autorest/go@latest', '--file-prefix="zz_generated_"', '--track2', '--go', '--debug', '--module-version=0.1.0'];
```
Comment (changlong-liu) [inline tools/sdk-testgen/packages/autorest.gotest/test/integrationtest/testAutorest.ts:49]: How about make some RP use '--module-version=0.1.0' and some don't?
Then we can cover both cases.

Comment (tadelesh) [reply tools/sdk-testgen/packages/autorest.gotest/test/integrationtest/testAutorest.ts:49]: I move the version num to the readme config. For now, two RP will use `0.1.0` and another will use `2.0.0-beta.1`.

**Review by changlong-liu — APPROVED** [review]

**Pull request merged**